### PR TITLE
Addition of the workflow_nodes for WorkflowRunJob

### PR DESCRIPTION
**Pull request by konono**
In the job.py file, I added a new property called workflow_nodes to the WorkflowJobRun class. This property is designed to retrieve information about the WorkflowNodes associated with a particular job run.

These changes appear to enhance the library's capabilities by allowing users to access WorkflowNodes information related to job runs and providing a dedicated class for modeling WorkflowNodes in Ansible Tower.

### CODE REVIEW

**Comment by costastf** [comment]
Hi @konono , thank you very much for this contribution! Would you mind moving the WorkflowNodes code under the job.py module and also add it in the relevant __init__.py since there is also a WorkflowJobRun class there and it feels like it fits. What do you think?

**Comment by costastf** [comment]
oh and please also add yourself on the Credits of the module and the CONTRIBUTING.rst in the root as a contributor.

**Comment by konono** [comment]
Hi @costastf , thank you your reply !
I have made corrections based on your suggestion, please review.

File: towerlib/entities/job.py [hunk 1]
```diff
@@ -1986,6 +1997,53 @@ def module_name(self):
...
+
+class WorkflowNodes(Entity):
+    """Models the user entity of ansible tower."""
+
+    def __init__(self, tower_instance, data):
+        Entity.__init__(self, tower_instance, data)
+
+    @property
+    def count(self):
+        """The job this event belongs to.
```
Comment (costastf) [inline towerlib/entities/job.py:2009]: Can we please update the docstrings to match the usage? Also on the other properties?

Comment (konono) [reply towerlib/entities/job.py]: Done.

**Comment by costastf** [comment]
💪

**Pull request merged**